encoder/vaapiencoder_base.cpp

```diff
@@ -450,6 +452,19 @@ const ProfileMapItem g_profileMap[] = {
     { PROFILE_H265_MAIN10, VAProfileHEVCMain10 },
 };
 
+#else
+
+const ProfileMapItem g_profileMap[] = {
+    { PROFILE_H264_BASELINE, VAProfileH264Baseline },
+    { PROFILE_H264_CONSTRAINED_BASELINE, VAProfileH264ConstrainedBaseline },
+    { PROFILE_H264_MAIN, VAProfileH264Main },
+    { PROFILE_H264_HIGH, VAProfileH264High },
+    { PROFILE_JPEG_BASELINE, VAProfileJPEGBaseline },
+};
+
+#endif
+
```

Comment:
Just need disable line 449 and 450 to check LIBVA_VERSION or __BUILD_H265ENC__,     
#ifdef __BUILD_H265ENC__
 { PROFILE_H265_MAIN, VAProfileHEVCMain },
  { PROFILE_H265_MAIN10, VAProfileHEVCMain10 },
#endif